### PR TITLE
consolidate manage cluster section

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/developer-experience/issues/241 by consolidating cluster actions into one page.

## Notes

**Will backport upon approval.**

NOTE -- this PR is **solely** within the scope of https://github.com/camunda/developer-experience/issues/241, and does _not_ address https://github.com/camunda/camunda-docs/issues/3199.

This PR only condenses the cluster actions into one page as a starting point.

Tomorrow, I will launch a Miro board to discuss the content and sidebar layout in full that comes with https://github.com/camunda/camunda-docs/issues/3199. CC @theburi 

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Review by conceptualshark — APPROVED** [review]
Love merging this into one section - it looks great to me. I ran it through locally for any missing links/updates and it all seems good. 👍

File: docs/components/console/manage-clusters/create-cluster-include.md [hunk 1]
```diff
@@ -40,3 +40,7 @@ Users without **Admin** roles can deploy only on `dev`, `test`, or `stage` clust
 ![cluster-healthy](./img/cluster-overview-new-cluster-healthy.png)
 
 3. After the cluster is created, click on the cluster name to visit the cluster detail page.
+
+:::note
+**Cluster auto-pause** is not yet available and only applies to non-Enterprise clusters. Development clusters will be paused if they go unused for two hours. When a cluster is paused, not all functionality will be limited, including the execution of BPMN timers and BPMN message catch events. To resume your cluster, navigate to the **Clusters** tab in Console, select the cluster name, and click **Resume cluster** within the **Cluster Details**.
```
Comment (ultraschuppi) [inline docs/components/console/manage-clusters/create-cluster-include.md:45]: >  When a cluster is paused, not all functionality will be limited, including the execution of BPMN timers and BPMN message catch events.

not sure what this means, times will continue to work? ;)

> Development clusters will be paused if they go unused for two hours

i don't think this is how it works

Comment (christinaausley) [reply docs/components/console/manage-clusters/create-cluster-include.md:45]: I just migrated over the existing content for this PR -- is the content outdated then? Do you know the timeframe of when clusters are paused?

Comment (christinaausley) [reply docs/components/console/manage-clusters/create-cluster-include.md:45]: @ultraschuppi Do you have any insight here I can update accordingly?

Comment (christinaausley) [reply docs/components/console/manage-clusters/create-cluster-include.md:45]: the current state of autopausing:

- clusters in trial organisations
- tagged with dev
- will be sent to sleep ~ 8h after they have been created / woken up the last time

we put trial-org clusters to sleep after a few days if we think they are not used (no process instances started), but i don't think this is even communicated

To resolve in a separate PR 👍

**Comment by christinaausley** [comment]
I'm going to resolve the conflicts here and backport, with plans to open a PR regarding https://github.com/camunda/developer-experience/issues/390 this week.

File: docs/components/console/manage-clusters/create-cluster-include.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/console/manage-clusters/create-cluster-include.md]: Also

> When a cluster is paused, not all functionality will be limited, including the execution of BPMN timers and BPMN message catch events.

> not sure what this means, times will continue to work? ;)

I've updated this wording @ultraschuppi 👍

**Comment by christinaausley** [comment]
This is ready for final review if you don't mind having a second look @conceptualshark 😄

**Review by conceptualshark — APPROVED** [review]
👍 Reviewed content, and double-checked the links. With the autopause updates going into a separate PR, this looks good to me!

**Comment by mesellings** [comment]
@christinaausley I see this work was merged (looks good btw!), unfortunately this puts the `Note` taken out as part of the[ Auto-pause](https://github.com/camunda/camunda-docs/pull/4085) work back into the docs, into the following two places:

Resume cluster:
image

At the bottom of the https://docs.camunda.io/docs/next/components/concepts/clusters/ page

How shall we handle this? Shall I create an issue and PR and take it out, or are you working on the cluster docs for any other issues atm?

**Comment by christinaausley** [comment]
@mesellings Feel free to remove this as you see fit based on your ongoing work 👍 Apologies for the overlap.

**Comment by mesellings** [comment]
Np - will do thanks! 👍

**Comment by mesellings** [comment]
Issue created to address this: https://github.com/camunda/camunda-docs/issues/4373

**Pull request merged**